cannon/README.md

```diff
@@ -42,19 +42,20 @@ make cannon
 ./bin/cannon run \
     --pprof.cpu \
     --info-at '%10000000' \
-    --proof-at never \
+    --proof-at '=<TRACE_INDEX>' \
+    --stop-at '=<STOP_INDEX>' \
+    --snapshot-at '%1000000000' \
     --input ./state.json \
     -- \
     ../op-program/bin/op-program \
     --network goerli \
```

Comment:
While we're here it probably makes sense to update this be `op-mainnet` since goerli has been shutdown now.